app/Services/CampaignService.php

```diff
@@ -40,9 +40,11 @@ public function find($id)
             $campaign = $this->phoenix->getCampaign($id);
 
             $this->cache->store($campaign['data']['id'], $campaign['data']);
+
+            $campaign = $campaign['data'];
         }
 
-        return $campaign['data'];
+        return $campaign;
```

Comment:
Think you can just return `return $campaign['data']` without storing it in a variable.